universal/result.py

```diff
@@ -393,14 +393,20 @@ def plot(
             res.plot(assets=assets, ax=ax1, **kwargs)
             ax2 = plt.subplot2grid((3, 1), (2, 0), sharex=ax1)
 
+            color = kwargs.get('color')
```

Comment:
Whatd do you think about moving `color` as optional arg to `plot` function? That would be more explicit than getting it from kwargs.

Reply:
Sure thing! Will push it right away.